app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_sign_up_params, only: [:create]
+
+  def create
+    resource = User::Create.call(user: build_resource(sign_up_params))
```

Comment:
I would pass just params as an argument. The controller knows about params and can have no idea what should be built from them. It a service object role to know what to build and how to do this.

👍 for use of key arguments